doc/vim-lsp.txt

```diff
@@ -49,6 +51,9 @@ CONTENTS                                                  *vim-lsp-contents*
       LspTypeDefinition                     |LspTypeDefinition|
       LspWorkspaceSymbol                    |LspWorkspaceSymbol|
     Mappings                              |vim-lsp-mappings|
+       <plug>(lsp-preview-close)            |<plug>(lsp-preview-close)|
```

Comment:
should we name this `lsp-floating-preview-close` to avoid confusion with vim preview?

Reply:
But it will close the "normal" preview just as well.

Reply:
What is with the name? I have no strong opinion, but it can be used to close the normal preview as well, so I don't know why it should be called floating-preview-close, but it's okay with me.

Reply:
If it closes normal preview then ok with the name.